apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-modal.tsx

```diff
@@ -0,0 +1,54 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { FC, useEffect, useState } from 'react';
+import { ThemeUpgradeModal } from 'calypso/components/theme-upgrade-modal';
+import { useOverrideSaveButton } from './hooks/use-override-save-button';
+
+import './upgrade-modal.scss';
+
+const GRIDICONS_URL =
+	'https://widgets.wp.com/wpcom-block-editor/images/gridicons-47c7fb356fcb2d963681.svg';
```

Comment:
This loads 404 for me (sandboxed):

<img width="283" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/05502070-9106-4af2-8d8a-90ffb8d0f0c7">